frontend/package.json

```diff
@@ -346,5 +346,6 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx,json,gql,graphql}": "eslint --color --fix"
-  }
+  },
```

Comment:
We're note using Corepack, so please remove.

Reply:
Could we open a ticket to add this in an another PR? I don't think this affects non-corepack installations of `yarn`. Corepack will also automatically add the `packageManager` field every time `yarn` is run, which gets annoying to remove

Reply:
@Mylanos looked in to us adding Corepack and learned Corepack is being *removed* from Node, so it seems like a futile exercise?

Reply:
TIL. Fair point, although rather depressing that corepack is being removed

Reply:
Removed it, every time I commit it's adding this to the current changes so I forgot to back it out. It's a shame it's being removed